src/test/java/org/example/UserTest.java

```diff
@@ -3,119 +3,118 @@
         Assert.assertTrue(errors.isEmpty(), errors.toString());
     }
 
     @Test
-    public void testGetUser() throws IOException, ParseException {
-        ClassicHttpResponse response = httpService.sendGetRequest(testDataFactory.getId());
-        int responseCode = response.getCode();
-        String responseBody = EntityUtils.toString(response.getEntity());
-        Assert.assertEquals(responseCode, HTTP_OK, "Wrong response code. Expected: " + HTTP_OK + " Received: " + responseCode);
-        List<ValidationMessage> errors = jsonValidator.validatePayload(responseBody, "getUserTemplate");
+    public void testGetUser() throws IOException, URISyntaxException, InterruptedException {
+        HttpResponse<String> response = httpService.sendGetRequest(testDataFactory.getId());
+        int responseCode = response.statusCode();
+        String responseBody = response.body();
+        Assert.assertEquals(responseCode, HttpStatus.SC_SUCCESS, "Wrong response code. Expected: " + HttpStatus.SC_SUCCESS + " Received: " + responseCode);
+        List<ValidationMessage> errors = jsonValidator.validateSchema(responseBody, "getUserTemplate");
         Assert.assertTrue(errors.isEmpty(), errors.toString());
     }
 
     @Test
-    public void testGetNonExistentUser() throws IOException {
-        ClassicHttpResponse response = httpService.sendGetRequest(NON_EXISTENT_ID);
-        int responseCode = response.getCode();
-        Assert.assertEquals(responseCode, HTTP_NOTFOUND, "Wrong response code. Expected: " + HTTP_NOTFOUND + " Received: " + responseCode);
+    public void testGetNonExistentUser() throws IOException, URISyntaxException, InterruptedException {
+        HttpResponse<String> response = httpService.sendGetRequest(NON_EXISTENT_ID);
+        int responseCode = response.statusCode();
+        Assert.assertEquals(responseCode, HttpStatus.SC_NOT_FOUND, "Wrong response code. Expected: " + HttpStatus.SC_NOT_FOUND + " Received: " + responseCode);
     }
 
     @Test
-    void testCreateUser() throws IOException, ParseException {
-        ClassicHttpResponse response = httpService.sendPostRequest(testDataFactory.generateTestUser());
-        int responseCode = response.getCode();
-        String responseBody = EntityUtils.toString(response.getEntity());
-        Assert.assertEquals(responseCode, HTTP_CREATED, "Wrong response code. Expected: " + HTTP_CREATED + " Received: " + responseCode);
-        List<ValidationMessage> errors = jsonValidator.validatePayload(responseBody, "postUserTemplate");
+    public void testCreateUser() throws IOException, URISyntaxException, InterruptedException {
+        User user = testDataFactory.generateTestUser();
+        HttpResponse<String> response = httpService.sendPostRequest(user);
+        int responseCode = response.statusCode();
+        String responseBody = response.body();
+        System.out.println(responseBody);
```

Comment:
I remind you once again that we do not use System.out.println(responseBody);, instead use Logger